deploy-manage/security.md

```diff
@@ -75,64 +74,186 @@ $$$maintaining-audit-trail$$$
+::::{note}
+As part of your overall security strategy, you can also do the following:
 
-:::{note}
-The availability and configurability of security features vary by deployment type. Refer to [Security by deployment type](#security-features-by-deployment-type) for a comparison table.
-:::
+* Prevent unauthorized access with [password protection and role-based access control](/deploy-manage/users-roles.md).
+* Maintain an [audit trail](/deploy-manage/security/logging-configuration/security-event-audit-logging.md) for security-related events.
+* Control access to dashboards and other saved objects in your UI using [Spaces](/deploy-manage/manage-spaces.md).
+* Connect a local cluster to a [remote cluster](/deploy-manage/remote-clusters.md) to enable [cross-cluster replication](/deploy-manage/tools/cross-cluster-replication.md) and [cross-cluster search](/solutions/search/cross-cluster-search.md).
+* Manage [API keys](/deploy-manage/api-keys.md) used for programmatic access to Elastic.
+::::
 
-## Security topics
+The availability and configurability of security features vary by deployment type. On every page, you'll see deployment type indicators that show which content applies to specific deployment types. Focus on sections tagged with your deployment type and look for subsections specifically addressing your deployment model.
 
-The documentation is organized into three main areas.
+At the end of this doc, there's also a [comparison table](#comparison-table) showing feature availability and configurability by deployment type.
 
-On every page, you'll see deployment type indicators that show which content applies to specific deployment types. Focus on sections tagged with your deployment type and look for subsections specifically addressing your deployment model.
+## Managed security in Elastic Cloud
+```yaml {applies_to}
+deployment:
+  ess: all
+serverless: all
+```
 
-### 1. Secure your orchestrator
+Elastic Cloud has built-in security. For example, HTTPS communications between Elastic Cloud and the internet, as well as inter-node communications, are secured automatically, and cluster data is encrypted at rest. 
 
-The [security of your orchestrator](security/secure-hosting-environment.md) forms the foundation of your overall security posture. This section covers environment-specific security controls:
+You can augment Elastic Cloud security features in the following ways:
+* Configure [traffic filtering](./security/traffic-filtering.md) to prevent unauthorized access to your deployments. [Elastic Cloud Hosted]
+* Encrypt your deployment with a [customer-managed encryption key](./security/encrypt-deployment-with-customer-managed-encryption-key.md). [Elastic Cloud Hosted]
+* [Secure your settings](./security/secure-settings.md) using {{es}} and {{kib}} keystores. [Elastic Cloud Hosted]
+* Use the list of [Elastic Cloud static IPs](./security/elastic-cloud-static-ips.md) to allow or restrict communications in your infrastructure.
```

Comment:
```suggestion
In {{ech}}, you can augment these security features in the following ways:
* Configure [traffic filtering](./security/traffic-filtering.md) to prevent unauthorized access to your deployments.
* Encrypt your deployment with a [customer-managed encryption key](./security/encrypt-deployment-with-customer-managed-encryption-key.md).
* [Secure your settings](./security/secure-settings.md) using {{es}} and {{kib}} keystores.
* Use the list of [Elastic Cloud static IPs](./security/elastic-cloud-static-ips.md) to allow or restrict communications in your infrastructure.
```